src/explore-education-statistics-admin/src/pages/publication/components/PublicationContactForm.tsx

```diff
@@ -36,16 +36,17 @@ const PublicationContactForm = ({
       .required('Enter a team email')
       .email('Enter a valid team email'),
     contactName: Yup.string().required('Enter a contact name'),
-    contactTelNo: Yup.string().test({
-      name: 'telephone',
-      message: 'Enter a contact telephone',
-      test: value => {
-        if (!value) {
-          return false;
-        }
-        return true;
-      },
-    }),
+    contactTelNo: Yup.string()
+      .trim()
+      .matches(
+        /^[0-9 \t]*$/,
+        'The telephone number should only contain numeric characters',
```

Comment:
This message is inconsistent with other validation messages and the recommended message style. It doesn't directly reference the field as labelled and doesn't mention that whitespace is allowed either.

`Contact telephone must only contain numeric or whitespace characters` might be better.